src/content/eth2/beacon-chain/index.md

```diff
@@ -36,29 +36,30 @@ Staking and becoming a validator is easier than [mining](/developers/docs/mining
 If you're interested in becoming a validator and helping secure the Beacon Chain, <a href="/eth2/staking/">learn more about staking</a>.
 </InfoBanner>
 
-This is also an important change for the second Eth2 upgrade: [shard chains](/eth2/shard-chains/).
+This is also an important change for another Eth2 upgrade: [shard chains](/eth2/shard-chains/).
 
 ### Setting up for shard chains {#setting-up-for-shard-chains}
 
-Shard chains will be the second Eth2 upgrade. They’ll increase the capacity of the network and improve transaction speed by extending the network to 64 blockchains. The Beacon Chain is an important first step in introducing shard chains, because they require staking to work securely.
+Shard chains will be an Eth2 upgrade that follows the merging of mainnet with the Beacon Chain. These "shards" will increase the capacity of the network and improve transaction speed by extending the network to 64 blockchains. The Beacon Chain is an important first step in introducing shard chains, because they require staking to work securely.
```

Comment:
This first sentence reads a little funny. Shard chains (plural)... an Eth2 upgrade (singular). 

Maybe just: "After mainnet merges with the Beacon Chain, the next upgrade will introduce shard chains to the Proof-of-stake network"